shim/test/test262/annexB/built-ins/Object/prototype/__defineGetter__/define-new.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
```

Comment:
I'm still studying the best pattern to bring test262 in. This is useful as is but most likely to evolve.